### DISCUSSION TITLE
juicefs/Auth error

### DISCUSSION THREAD

**Discussion by vivekcode101** [juicedata/juicefs#5291] [Q&A]
Hello Folks, recently found about juicefs and was trying to use S3 bucket as PVC but when I am creating the pod the juicefs csi controller is giving me auth error
![image](https://github.com/user-attachments/assets/69ffde58-b66b-461f-9ae3-eddd6f8e3aba)

I am using kubeadm's control plane on my local system and calico as my network.
I've been trying to resolve the issue but unable to do so.

**Comment by zxh326** [comment]
check our cloud service(`juicefs.com`) is accessible within the cluster?
